templates/tab/ts/default/package.json

```diff
@@ -5,6 +5,9 @@
     "dependencies": {
         "@fluentui/react-northstar": "^0.54.0",
         "@microsoft/microsoft-graph-client": "^3.0.0",
```

Comment:
I tried the template, we need to remove `"@microsoft/microsoft-graph-client": "^3.0.0"` reference and `tabs\src\components\sample\lib\useGraph.ts` file, otherwise, there will be a conflict when compile

Reply:
Right, MGT is still on 2.x and TeamsFx uses 3.x. We're working on making it happen from the MGT side.

Reply:
Do we need to remove `useGraph.ts` in this case? This could probably run with a previous version of the Graph SDK anyway.

Reply:
Yes, I think we can remove it because it is not used anymore